tomlkit/items.py

```diff
@@ -957,6 +984,15 @@ def __init__(self, value: list, trivia: Trivia, multiline: bool = False) -> None
         self._multiline = multiline
         self._reindex()
 
+    def unwrap(self, recursive: bool = True) -> str:
```

Comment:
Are there any cases that we don't want recursive unwrap? `list(array)` should do the trick.

Reply:
So you mean just returning `list(self)`?

> Are there any cases that we don't want recursive unwrap?

Is that only applicable to the `items.Array` object or do you want me to make all of of the `unwrap` methods recursive?

Reply:
Ok, I think I see something of what you're saying. 

So if a create an `Array` using `array = items.Array([item(2), item(2.5), item(False)], trivia])`, the third one (boolean) gets turned into a ppo but not the others

This makes the recursive `unwrap` fail on that

But if I do `return list(self)` then the `Integer` and `Float` items in indices 0 and 1 don't come out as ppo

Reply:
I mean `recursive=False` is equal to `list(array)`, so we don't need this optional flag and make all `unwrap` recursive.